### PR TITLE
Instance type for qesap regression

### PR DESCRIPTION
**Pull request by mpagot**
Add a config.yaml variable and a setting to control the VM type for the Hana machines

- Related ticket: TEAM-7486
- Verification run: 

SAPCONF : http://openqaworker15.qa.suse.cz/tests/135376 

SAPTUNE : http://openqaworker15.qa.suse.cz/tests/135375 The generated .tfvars is fine http://openqaworker15.qa.suse.cz/tests/135375/file/configure-terraform.tfvars

### CODE REVIEW

File: tests/sles4sap/qesapdeployment/configure.pm [hunk 1]
```diff
@@ -30,6 +30,7 @@ sub run {
     $variables{SSH_KEY_PRIV} = '/root/.ssh/id_rsa';
     $variables{SSH_KEY_PUB} = '/root/.ssh/id_rsa.pub';
     $variables{SCC_REGCODE_SLES4SAP} = get_required_var('SCC_REGCODE_SLES4SAP');
+    $variables{HANA_INSTANCE_TYPE} = get_var('QESAP_HANA_INSTANCE_TYPE', 'r6i.xlarge');
```
Comment (mpagot) [inline tests/sles4sap/qesapdeployment/configure.pm:33]: The default value match this one https://github.com/SUSE/qe-sap-deployment/blob/accc817d3cd1d606afa65c136f18b432bff2797f/terraform/aws/variables.tf#L191

Comment (lilyeyes) [reply tests/sles4sap/qesapdeployment/configure.pm:33]: Should the default value 'default = "r6i.xlarge"' be changed according to https://jira.suse.com/browse/TEAM-7291?

**Review by lilyeyes — APPROVED** [review]
LGTM.

File: tests/sles4sap/qesapdeployment/configure.pm (hunk 1 above)
Comment (BillAnastasiadis) [reply tests/sles4sap/qesapdeployment/configure.pm:33]: Probably the values mentioned in the ticket are to be contained in `QESAP_HANA_INSTANCE_TYPE` var, and default to be used only should this var is not specified

**Review by BillAnastasiadis — APPROVED** [review]
LGTM

File: tests/sles4sap/qesapdeployment/configure.pm (hunk 1 above)
Comment (mpagot) [reply tests/sles4sap/qesapdeployment/configure.pm:33]: yes, this PR is just the first step to be able to create a JobGroup testing all the types

**Pull request merged**